### DISCUSSION TITLE
what does the new preset mean in the last update?

### DISCUSSION THREAD

**Discussion by MohammadAboulEla** [lllyasviel/Fooocus#2562] [Q&A] [answered]
it is not clear how we use them and it does not make sense if I accidentally clicked one of them it start downloading things I do not need! and why not simply! add a custom button for average/advanced users to simply enter the Sampling Step and Guidance Scale! and Sampling Method! most of user has become familiar with these settings.
and I know I can change them in Developer Debug Mode but why should I go dig down deep to change a simple step slider that should be at least in the advanced tap.

**Comment by mashb1t** [comment, marked as answer]
Presets are described in the readme and offer presets for settings you'd manually have to set in the UI. They offer a very simple and straight forward set of configs with the best features for each one. Presets have been in Fooocus since almost the beginning and are now able to be selected in the UI.
Fooocus aims for simplicity, which is why "add a custom button for average/advanced users" is already done in developer debug mode, as well as much more advanced features. This can also be seen as a "You know what you're doing" area.
